<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeLoginWidget</class>
 <widget class="QWidget" name="DeLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeLoginWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>140</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border:1px solid gray;
border-radius:5px;
min-height:30px;
font: 16pt &quot;新宋体&quot;;
}

QLabel
{
font: 14pt &quot;新宋体&quot;;
min-width:60px;
}

QGroupBox {
     border: 1px solid gray;
     border-radius: 3px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 10px;
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }

QPushButton {
    
	font: 16pt &quot;新宋体&quot;;
	background-color: rgb(244, 244, 244);
     border: 1px solid #8f8f91;
     border-radius: 6px;
     min-width: 100px;
     min-height: 30px;
 }

 QPushButton:pressed {
	background-color: rgb(255, 255, 255);
 }

</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">max-width:280px;
max-height:180px;</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="okBtn">
              <property name="maximumSize">
               <size>
                <width>282</width>
                <height>182</height>
               </size>
              </property>
              <property name="text">
               <string>登录</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="cancelBtn">
              <property name="maximumSize">
               <size>
                <width>282</width>
                <height>182</height>
               </size>
              </property>
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QGroupBox" name="groupBox">
              <property name="maximumSize">
               <size>
                <width>302</width>
                <height>208</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="title">
               <string>登录</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout">
                 <property name="horizontalSpacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>用户:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="userLineEdit"/>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_6">
                 <property name="horizontalSpacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>密码:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="passwdLineEdit">
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="DeVirtualKeyboradWidget" name="keyBoradWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeVirtualKeyboradWidget</class>
   <extends>QWidget</extends>
   <header location="global">devirtualkeyboradwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
